src/components/Searchbar.js

```diff
@@ -26,6 +27,7 @@ function Search() {
     }
   }, [search]);
```

Comment:
I'm getting a warning here. If eslint is to be believed, I think the preferred approach is to include both search and setSearch in the useEffect dependencies

**Compiled with warnings.**

./src/components/Searchbar.js
  Line 28:6:  React Hook useEffect has a missing dependency: 'setSearchResults'. Either include it or remove the dependency array  react-hooks/exhaustive-deps